tests/src/org.apache.commons/commons-dbcp2/2.9.0/src/test/java/org_apache_commons/commons_dbcp2/CommonDbcp2Test.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_apache_commons.commons_dbcp2;
```

Comment:
Tests from this class require only the following metadata:

- reflect-config.json
```
[
  {
    "name":"org.apache.commons.dbcp2.PoolingDriver"
  }
]
```
- resource-config.json
```
{
    "bundles": [
        {
            "name": "org.apache.commons.dbcp2.LocalStrings"
        }
    ]
}
```
So please modify metadata in the `metadata/org.apache.commons/commons-dbcp2/2.9.0` folder or add more tests which would require other metadata.

Also I am not sure that `org.apache.commons.dbcp2.PoolingDriver` should be added to the `reflect-config.json` file in the main metadata directory since `Class.forName("org.apache.commons.dbcp2.PoolingDriver");` is called in the test not in the `commons-dbcp2` library itself, so that metadata entry should probably go to the `tests/src/org.apache.commons/commons-dbcp2/2.9.0/src/test/resources/META-INF/native-image/dbcp2-test-only/reflect-config.json` file. @vjovanov - your thoughts?